sycl/source/detail/graph_impl.cpp

```diff
@@ -1775,6 +1844,208 @@ void dynamic_parameter_base::updateAccessor(
   impl->updateAccessor(Acc);
 }
 
+void dynamic_parameter_impl::updateValue(const raw_kernel_arg *NewRawValue,
+                                         size_t Size) {
+  // Number of bytes is taken from member of raw_kernel_arg object rather
+  // than using the size parameter which represents sizeof(raw_kernel_arg).
+  std::ignore = Size;
+  size_t RawArgSize = NewRawValue->MArgSize;
+  const void *RawArgData = NewRawValue->MArgData;
```

Comment:
Since we end up with a size and ` const void*` from the raw arg data, can this function not be implemented by forwarding to the other overload? Like so:

```c++
  size_t RawArgSize = NewRawValue->MArgSize;
  const void *RawArgData = NewRawValue->MArgData;

  updateValue(RawArgData, RawArgSize);
```

Reply:
Top review feedback! We absolutely can - updated